Robot Code/src/org/usfirst/frc/team3316/robot/subsystems/Hood.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team3316.robot.subsystems;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.robotIO.DBugSpeedController;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+
+public class Hood extends DBugSubsystemCC
+{
+	private DBugSpeedController hoodMotor;
+	private AnalogPotentiometer hoodPot;
+
+	public Hood()
+	{
+		hoodMotor = Robot.actuators.hoodMotor;
+		addSpeedController(hoodMotor);
+
+		hoodPot = Robot.sensors.hoodPot;
+	}
+
+	public void initDefaultCommand()
+	{
+	}
+
+	/**
+	 * Set the voltage for the motors of this subsystem.
+	 * @par v - The voltage for the motors (between 1.0 to -1.0).
+	 */
+	public boolean setMotors(double v)
+	{
+		// If the hood is able to move.
+		if (getAngle() > (double) config.get("hood_Pot_LowThresh")
+				&& getAngle() < (double) config.get("hood_Pot_HighThresh")) {
+			return super.setMotors(v);
+		}
+		
+		super.setMotors(0);
+		return false;
```

Comment:
Might be useful to add some sort of debug message - in case we run into issues with unexplained movement of the hood. something along the lines of: The first time you reach the `else` of this statement print to log a message saying we've reached the limit, and reset this message each time you are back in valid location.